CHANGELOG.md

```diff
@@ -1,21 +1,10 @@
-# NEXT RELEASE
-
-### Enhancements
-* None.
+# 6.0.10 Release notes
 
 ### Fixed
 * Re-enable compilation using SSE (since v6.0.7)
 * Risk of aborting from Table::migrate_indexes() reduced.
```

Comment:
How could this be experienced for an end user?

Reply:
To me 'aborting' means that the program crashes. Should this be spelled out more explicitly?